src/main/resources/application.properties

```diff
@@ -4,5 +4,7 @@ spring.datasource.username=root
 spring.datasource.password=---SQLdno12
```

Comment:
let's not push username/passwords and other credentials or sensitive information to remote git, leave it blank or use some placeholder instead